code/game/objects/items/devices/multitool.dm

```diff
@@ -167,6 +178,7 @@
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "multitool"
 	toolspeed = 0.1
+	cell_override = /obj/item/stock_parts/cell/bluespace //SKYRAT EDIT ADDITION
```

Comment:
Assuming here you have the ability to make cells non-removable.
If so, replace this with a void core, or something similar.